_includes/docs_category_nav.html

```diff
@@ -0,0 +1,53 @@
+<div class="left-sidebar">
```

Comment:
I'm sorry I don't follow what you mean by "site"? If you mean position within the layout, then this is the logic that builds the navigation in the left-sidebar of the docs page layout. So all docs pages use this include by way of the docs layout (_layouts/docs.html).